src/declarations/types.d.ts

```diff
@@ -0,0 +1,28 @@
+import interfaces from "./interfaces";
+
+export declare type Step = {
+  message: string;
+  action: interfaces.StepAction;
+};
+
+export declare type ValidationResult = {
```

Comment:
in case you missed it, seems like `@types/validate-npm-package-name` exists that can install as a devdep

Reply:
No, I didn't miss it. I couldn't really spend that much time then so I first thought of implementing more necessary changes.